simplq/src/components/pages/Admin/admin.module.scss

```diff
@@ -8,6 +8,26 @@
   height: max-content;
 }
 
+.share {
+  .button-background {
+    background-color: $primary-color-dark;
+    border-radius: 0.6rem !important;
+  }
+
+  .popper {
+    z-index: 100;
+    background-color: #fff;
+  }
+
+  .share-button {
+    display: flex;
+  }
+
+  .share-icon {
+    margin-right: 10px;
```

Comment:
can you change to rem here. @maaverik likes rems. 😉

Reply:
Ok sure :grinning: